backend/src/services/response.ts

```diff
@@ -25,6 +25,23 @@ export async function getResponsesByParentIds(_user: UserInterface, parentIds: s
   return responses
 }
 
+export async function getResponseById(user: UserInterface, responseId: string) {
+  const response = await ResponseModel.findOne({ _id: responseId })
+
+  if (!response) {
+    throw NotFound(`The requested response was not found.`, { responseId })
+  }
+
+  if (response.entity !== toEntity('user', user.dn)) {
+    throw Forbidden('Only the original author can update a comment or review response.', {
+      userDn: user.dn,
+      responseId,
+    })
+  }
```

Comment:
This suggests that you can only get individual responses on responses that you've written. Is this expected?

Reply:
Function has been removed